### PR TITLE
Update document

### PR DESCRIPTION
**Pull request by 10sr**
* Add documents about two customize variables
* Add editorconfig customize group under tools (make custom variables accessible via M-x customie -> Programming -> Tools)
* Make some other minor changes

ref: #79

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,39 @@ in to all different packages for whitespace trimming to inform
...
+### `editorconfig-indentation-alist`
+
+Alist of indentaion setting mothods by modes.
+
+For the easiest case to add a new support for a major-mode, you just need to
+add a pair of major-mode symbol and its indentation variables:
+
+```emacs-lisp
+(add-to-list 'editorconfig-indentation-alist
+    ;; Just an example, of course EditorConfig already includes this setting!
```
Comment (xuhdev) [inline README.md]: Just one minor issue: "already includes" to "has already included" :)

Comment (10sr) [inline README.md]: Thanks! :cake:

**Comment by xuhdev** [comment]
Do you think it'll be a good idea to define a few functions to alter `editorconfig-indentation-alist`? Then we probably can replace the web-mode example with a more elegant one, i.e., removing the entry in the alist.

**Comment by 10sr** [comment]
I think it would be a good idea :+1:.
However, I also think it is better not to hide `editorconfig-indentation-alist` from users, because alists is a common data structure for emacs lisp users (and lispers as well, I think) and hiding this variable can bring an unnecessary complexity.

**Pull request merged**